### PR TITLE
Changed default VARDEFN search path and trimmed document blocks

### PR DESCRIPTION
**Pull request by ywangwof**
List of changes you can decide whether to accept or not,

1. README file format by adding two blank spaces for new lines in the **Note** section.
2. Default search path for `var_defn.sh` will be `../var_defn.sh`.
3. Make **get_files.sh** works the same way as **run_fv3lam.sh** by adding options `-h`, `-n`, `-v`, `-m` etc.
4. Determine whether to submit job script or run it based on whether the file `$EXPTDIR/FV3LAM_wflow.xml` exists or not, but not on the machine as before. The logic will be much simpler.
5. I do not understand why you want to 
 ```
         touch out.${tasknames[$task]}_${jobid}
         touch err.${tasknames[$task]}_${jobid}
```
These two files should be created by the SLURM scheduler when the job is run. Furthermore, touching them in the current directory (script directory) makes no sense.

### CODE REVIEW

**Comment by climbfuji** [comment]
> List of changes you can decide whether to accept or not,
> 
> 1. README file format by adding two blank spaces for new lines in the **Note** section.
> 2. Default search path for `var_defn.sh` will be `../var_defn.sh`.
> 3. Make **get_files.sh** works the same way as **run_fv3lam.sh** by adding options `-h`, `-n`, `-v`, `-m` etc.
> 4. Determine whether to submit job script or run it based on whether the file `$EXPTDIR/FV3LAM_wflow.xml` exists or not, but not on the machine as before. The logic will be much simpler.
> 5. I do not understand why you want to
> 
> ```
>         touch out.${tasknames[$task]}_${jobid}
>         touch err.${tasknames[$task]}_${jobid}
> ```
> 
> These two files should be created by the SLURM scheduler when the job is run. Furthermore, touching them in the current directory (script directory) makes no sense.

We can remove the "touch" statements, but your code is behind, because the current code touches these files in ${LOGDIR}. Please pull in the updates from master and resolve any conflicts first. The reason for touching was just lazyiness, because I can then just do copy&paste and run tail -f ... and see when it starts up. But no problem to do copy&paste, run touch first m and then tail -f.

But please pull in the updates from origin/master.

**Comment by ywangwof** [comment]
Ok. Fortunately, there is no conflict. Try again.

File: run_fv3lam.sh [hunk 1]
```diff
@@ -251,13 +257,11 @@ if [[ $show -ne 1 ]]; then
         echo "    ${LOGDIR}/out.${tasknames[$task]}_${jobid}"
         echo "    ${LOGDIR}/err.${tasknames[$task]}_${jobid}"
         echo " "
-        touch out.${tasknames[$task]}_${jobid}
-        touch err.${tasknames[$task]}_${jobid}
-    elif [[ $machine = "odin" ]]; then
-        sbatch ${jobscript}
+        #touch out.${tasknames[$task]}_${jobid}
```
Comment (climbfuji) [inline run_fv3lam.sh:260]: I would remove these two commented lines.

**Review by climbfuji — APPROVED** [review]
Looks all good, I assume you tested this on Stampede?

**Comment by ywangwof** [comment]
It was tested on my Macbook and Odin. I cannot run job on Stampede, but the script does not change anything about the job configurations. I only tested the script options on Stampede but did not actually run them.

**Comment by climbfuji** [comment]
> It was tested on my Macbook and Odin. I cannot run job on Stampede, but the script does not change anything about the job configurations. I only tested the script options on Stampede but did not actually run them.

Thanks, I'll go ahead and merge it. If there are any unexpected problems, we can fix them in a follow-up PR. I assume we need to update the wiki as well, are you going to do that?

**Comment by ywangwof** [comment]
The Wiki still works. I see no need for a change.

**Comment by climbfuji** [comment]
> The Wiki still works. I see no need for a change.

Ok, great.

**Comment by climbfuji** [comment]
I can confirm this still works fine on Stampede. Thanks for making the changes.

**Pull request merged**